plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -165,6 +179,24 @@ public boolean getSastEnabled() {
 		return false;
 	}
 
+	public String getIssueDescription(String issueId) {
+		if(StringUtils.isEmpty(issueId)) {
+			SnykLogger.logInfo("issueId is empty");
+			return "";
+		}
+		CompletableFuture<Object> issueDescription = executeCommand("snyk.generateIssueDescription", List.of(issueId));
```

Comment:
question: can't we use `CompletableFuture<String>`? Or doesn't the interface allow it?

Reply:
inter doesn't allow it